src/utils.jl

```diff
@@ -138,10 +138,10 @@ end
 
 # because isposdef keeps giving the wrong answer for samples
 # from Wishart and InverseWisharts
-hasCholesky(a::Matrix{Float64}) = isa(trycholfact(a), Cholesky)
+hasCholesky(a::Matrix{Float64}) = isa(trycholesky(a), Cholesky)
 
 function trycholfact(a::Matrix{Float64})
```

Comment:
Don't you need to change the name `trycholfact` to `trycholesky` here too if you are going to change it in line 141?

Reply:
Yeap! Note: a test for `trycholesky` should be added.